rmagick.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake-compiler', '~> 1.0'
   s.add_development_dependency 'rspec', '~> 3.8'
   s.add_development_dependency 'rspec_junit_formatter', '~> 0.4.1'
-  if RUBY_PLATFORM !~ /mswin|mingw/
+  if RUBY_PLATFORM !~ /mswin/
```

Comment:
It's OK to delete this line because I copied and pasted this for Windows platform (MinGW) on CI.

Reply:
@Watson1978 

Hello.

Ok, I know that in the past Windows mswin builds weren't publicly available, but I've now got a master build available (I created it for stdlib CI use since ruby/ruby does mswin testing).

As mentioned, jaro_winkler (RuboCop dependency) will not compile on mswin.  Hence, ok to leave it?

Reply:
Thank you for your advice. no problem to keep it :) 
(I've only performed simple installation tests on mswin)